### DISCUSSION TITLE
Rotation not work in the google model viewer

### DISCUSSION THREAD

**Discussion by samuelss1102** [community/community#20882] [Mobile] [answered]
On the desktop, the model rotation works as well.
But on the Mobile, it does not work vertically.
Is there anyone who knows the reason?

**Comment by savvydev007** [comment, marked as answer]
problem in the touch action 
please try this.
<model-viewer
      touch-action='pan-x, pan-y'
      src="/scene.glb"
      poster="/loading.svg"
      alt='A 3D model'
      shadow-intensity='1'
      camera-controls
      auto-rotate
      ar
      ar-scale="auto"
      quick-look-browsers="safari chrome"
      ar-modes='webxr scene-viewer quick-look'
    ></model-viewer>

**Reply by Farshid912** [reply]
> problem in the touch action please try this.
